.gitmodules

```diff
@@ -2,3 +2,6 @@
 	path = third_party/cutlass
 	url = https://github.com/NVIDIA/cutlass.git
 	branch = v2.4.0
+[submodule "third_party/fio"]
+	path = third_party/fio
+	url = https://github.com/axboe/fio.git
```

Comment:
Which version?

Reply:
master branch, added in gitmodules

Reply:
For FIO, whether it is better to choose one release version?

Reply:
They use tag for release, no branch. So we have to use a fixed commit for submodule.